avoid having to build from cloned Git repository by adding simulator/build_info.h file,
which was generated using "PETSC_DIR=/foo make build_info.h" in Git repo that corresponds with openCARP v3.2 sources
author: Kenneth Hoste (HPC-UGent)
--- openCARP-v3.2.orig/simulator/CMakeLists.txt	2020-08-11 13:55:42.774794926 +0200
+++ openCARP-v3.2/simulator/CMakeLists.txt	2020-08-11 13:58:39.811085211 +0200
@@ -3,12 +3,6 @@
 #######################################################################
 #                                carp                                 #
 #######################################################################
-add_custom_command(OUTPUT "build_info.h"
-    COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/build_info.py
-    DEPENDS build_info.py
-    COMMENT "Generate a C header file defining some information about the build"
-    VERBATIM
-)
 
 set(CARP_FILES "carp_d.h" "carp_p.cc" "carp_p.h")
 add_custom_command(OUTPUT ${CARP_FILES}
--- openCARP-v3.2.orig/simulator/build_info.h	1970-01-01 01:00:00.000000000 +0100
+++ openCARP-v3.2/simulator/build_info.h	2020-08-11 13:07:56.869383000 +0200
@@ -0,0 +1,12 @@
+#ifndef __BUILD_INFO__
+#define __BUILD_INFO__
+
+#define GIT_COMMIT_HASH      "f9fcd8bf495879e0f9145366198467158cf9ad0a"
+#define GIT_COMMIT_COUNT     834
+#define GIT_PATH             "https://git.opencarp.org/openCARP/openCARP.git"
+
+#define SUBREPO_REVISIONS ""
+
+#define SUBREPO_COMMITS ""
+
+#endif
